Makefile

```diff
@@ -5,14 +5,24 @@ FULL_GOPATH=${GOPATH}/src/${FULL_NAME}
 
 default: test
 
-get:
-	go get "github.com/omeid/go-resources/cmd/resources"
+dependencies:	
 	go get -t -v ./...
 
-build: get
-	cd example/agent-ping && resources -output="resources.go" -var="Resources" -trim="../" resources/* ../schema/*
-	cd example/agent-pong && resources -output="resources.go" -var="Resources" -trim="../" resources/* ../schema/*
+build: dependencies	
 	go build -v ./...
 
+examples: dependencies
```

Comment:
To me this doesn't work. If you do the following, agent-ping will fail to run.
$ make
$ agent-ping
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x7b8d2f]

goroutine 1 [running]:
main.MustOpenResources(0x8a8dc7, 0x18, 0xc420060058, 0x0, 0x7b89e5)
        /home/jp/workspace/perso/go/src/github.com/crucibuild/sdk-agent-go/example/agent-ping/pingAgent.go:38 +0x2f
main.NewPingAgent(0x0, 0xc4200e3f68)
        /home/jp/workspace/perso/go/src/github.com/crucibuild/sdk-agent-go/example/agent-ping/pingAgent.go:56 +0x5b
main.main()
        /home/jp/workspace/perso/go/src/github.com/crucibuild/sdk-agent-go/example/agent-ping/main.go:19 +0x22